scripts/download.sh

```diff
@@ -8,6 +8,12 @@ URL="$(echo $CTLSTORE_BOOTSTRAP_URL | sed -e s,$PREFIX,,g)"
 BUCKET="$(echo $URL | grep / | cut -d/ -f1)"
```

Comment:
this script is approaching "should be a go binary with tests" level of complexity, let me know what ya'll think

Reply:
Yah, I think we can change to that, but prefer not to block shipping this out on that.  Our initial intended use-case is only integrations-consumer which our team owns, so if there's any issue with this we can fix and deploy without it having a broad impact.

i.e., I'd lean towards velocity here and getting the change out into the problematic environment, then we can fix this portion up later. e.g., we can bring it into the reflector code proper in Go.